nodes/nodes-about-autoscaling-nodes.adoc

```diff
@@ -0,0 +1,78 @@
+include::modules/attributes-openshift-dedicated.adoc[]
+[id="nodes-about-autoscaling-nodes"]
+= About autoscaling nodes on a cluster
+:context: nodes-about-autoscaling-nodes
+toc::[]
+
+ifdef::openshift-dedicated[]
+[IMPORTANT]
+====
+Autoscaling is only available on clusters that were purchased through the Red Hat Marketplace.
+====
+endif::[]
+
+The autoscaler option can be configured to automatically scale the number of machines in a cluster.
+
+The cluster autoscaler increases the size of the cluster when there are pods that failed to schedule on any of the current nodes due to insufficient resources or when another node is necessary to meet deployment needs. The cluster autoscaler does not increase the cluster resources beyond the limits that you specify.
+
+Additionally, the cluster autoscaler decreases the size of the cluster when some nodes are consistently not needed for a significant period, such as when it has low resource use and all of its important pods can fit on other nodes.
+
+When you enable autoscaling, you must also set a minimum and maximum number of worker nodes.
+
+[NOTE]
+====
+Only cluster owners and organization admins can scale or delete a cluster.
+====
+
+[id="nodes-enabling-autoscaling-nodes"]
+== Enabling autoscaling nodes on a cluster
+
+Autoscale worker nodes on a cluster to increase or decrease the number of resources available. Autoscaling for worker nodes is configured in the machine pool definition if you are configuring an existing cluster.
```

Comment:
s/Autoscale worker nodes/You can enable autoscaling on worker nodes  Or something like that.

Reply:
_if you are configuring an existing cluster_
What if you are creating a new cluster? Can you set this option? Or, need to deploy the cluster then edit the machine pool definition?

Reply:
I wonder if you could combine these two sentences into one? 

_You can enable autoscaling on worker nodes to increase or decrease the number of resources available by editing the machine pool definition for an existing cluster._

Reply:
Sorry to pile on. For OCP, the term _resource_ seems to me to apply to CPU and memory. In that perspective, it seems that autoscaling would increase the CPU or memory. _increase or decrease the number of nodes_?